staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -280,6 +285,17 @@ var (
 		[]string{"code_path"},
 	)
 
+	watchListLatencies = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Subsystem:      APIServerComponent,
+			Name:           "watch_list_duration_seconds",
+			Help:           "Response latency distribution in seconds for watch list requests broken by group, version, resource and scope.",
+			Buckets:        []float64{0.1, 0.25, 0.5, 1.0, 2, 5, 10, 20, 30, 45},
```

Comment:
I still think that we want to use more buckets, and more aligned request_latencies metric which are:
```
{0.005, 0.025, 0.05, 0.1, 0.2, 0.4, 0.6, 0.8, 1.0, 1.25, 1.5, 2, 3, 4, 5, 6, 8, 10, 15, 20, 30, 45, 60
```

We can probably drop some values, but especially given we will have progress-notify eventually, I would like to have better comparison with lists, so I suggest:
```
{0.05, 0.1, 0.2, 0.4, 0.6, 0.8, 1.0, 2, 4, 6, 8, 10, 15, 20, 30, 45, 60}
```

Reply:
sure, I guess we are about to find out what values really matter, i don't have strong opinions here.